.mvn/maven.config

```diff
@@ -0,0 +1,2 @@
+-Pconsume-incrementals
```

Comment:
Hi,

please clarify why the usage of incrementals is necessary.
For the bom file only?

Reply:
it allows testing pre-release versions across components, and produces a release candidate automatically.

Used often in PCT (compatibility) testing when changes like this aren't merged.

it's the default for all new plugins, https://github.com/jenkinsci/archetypes/blob/master/common-files/.mvn/maven.config

Reply:
in this specific context, what are the consumed pre-release artifacts?

Reply:
If it is not necessary right now, please remove it. It can be added later if required.

Reply:
Have you ever heard the saying "[don't look a gift horse in the mouth](https://en.wiktionary.org/wiki/don%27t_look_a_gift_horse_in_the_mouth)"? Updating plugin build metadata is a chore, and here I have offered it to your plugin for free…